runtime/src/iree/builtins/ukernel/arch/arm_64/CMakeLists.txt

```diff
@@ -56,7 +57,9 @@ iree_link_bitcode(
 
 )
 
-endif()  # IREE_BUILD_COMPILER AND IREE_TARGET_BACKEND_LLVM_CPU
+else()  # _IREE_UKERNEL_BITCODE_BUILD_ARM_64
+file(WRITE "${CMAKE_CURRENT_BINARY_DIR}/ukernel_bitcode_arm_64.bc" "")
+endif()  # _IREE_UKERNEL_BITCODE_BUILD_ARM_64
```

Comment:
Do we still want to write this if `IREE_BUILD_COMPILER AND IREE_TARGET_BACKEND_LLVM_CPU` is false? Seems like clutter...

Reply:
Good point, done.